src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -670,11 +673,46 @@ public class DeleteOldRevisionsResult
 
             var deleted = DeleteRevisionsInternal(context, table, lowerIdPrefix, collectionName, changeVector, lastModifiedTicks, result.PreviousCount, revisionsToDelete, result);
 
+
+            NotifyAboutConflictRevisionsIfNeeded(conflicted, nonPersistentFlags, deleted, lowerIdPrefix);
+
             IncrementCountOfRevisions(context, lowerIdPrefix, -deleted);
             result.Remaining = result.PreviousCount - deleted;
             return result;
         }
 
+        private void NotifyAboutConflictRevisionsIfNeeded(bool conflicted, NonPersistentDocumentFlags nonPersistentFlags, long deleted, Slice lowerIdPrefix)
+        {
+            var lowerId = lowerIdPrefix.ToString();
+            lowerId = lowerId.Substring(0, lowerId.Length - 1); // cut the prefix last char (SpecialChars.RecordSeparator)
+
+            if (conflicted &&
+                nonPersistentFlags.Contain(NonPersistentDocumentFlags.ByEnforceRevisionConfiguration) == false &&
+                nonPersistentFlags.Contain(NonPersistentDocumentFlags.FromSmuggler) == false &&
+                deleted > 0)
+            {
+                var key = $"ConflictRevisionExceededMax/{lowerId}";
```

Comment:
this will generate a lot of unique notifications. Check how we do that for Paging notification for example. You need to aggregate.

Reply:
Done